app/controllers/deploys_controller.rb

```diff
@@ -49,6 +49,87 @@ def recent
     end
   end
 
+  # little search endpoint.  
+  # takes the same tokens that are used by the filter stuff
+  # on the recent deploys pages.  returrns a paginated 
+  # json object or CSV of stuff that people are interested in rather than 
+  # doing client side filtering.
+  # params:
+  #   * username (name of the user that started the job(s)
```

Comment:
changed.  I was just sorta copying what was in the recent deploy UI, but agree that it's confusing.